src/cursor/cur_ds.c

```diff
@@ -453,7 +453,9 @@ __wt_curds_open(WT_SESSION_IMPL *session, const char *uri, WT_CURSOR *owner, con
       __wt_cursor_notsup,                             /* cache */
       __wt_cursor_reopen_notsup,                      /* reopen */
       __wt_cursor_checkpoint_id,                      /* checkpoint ID */
-      __curds_close);                                 /* close */
+      __curds_close,                                  /* close */
+      __wt_cursor_get_details                         /* get_details */
```

Comment:
Thanks I believe I have done this correctly, but thought ENOTSUP was better for cursor types that do not yet support the call.

Reply:
I agree that ENOTSUP is the right return

But I haven't seen the corrected code, have you pushed it?  I think the case that would fail would be to create some kind of "other" cursor, like a backup cursor or metadata cursor and then call cursor.get_details on it.  The place where the code falls down is:
```
    cbt = (WT_CURSOR_BTREE *)cursor;
    if (cbt->checkpoint_txn) {
      ...
```
because in those other cursor types, `cursor` will not be a `(WT_CURSOR_BTREE *)`.  Perhaps I'm missing something.

You'll want to have a test case like that.